modules/hcp-dc-image-mirror.adoc

```diff
@@ -95,7 +95,7 @@ mirror:
 # ...
 ----
 +
-<1> Replace `4.x.y-build` with the supported {product-title} version you want to use.
+<1> Replace `<4.x.y-build>` with the supported {product-title} version you want to use.
```

Comment:
In the YAML example, be sure to add angle brackets around `4.x.y-build` to match the text in the annotation.